src/NotificationTemplate.php

```diff
@@ -273,11 +273,15 @@ public function getTemplateByLanguage(
                     $css = $this->fields['css'] ?? '';
 
                     $lang['content_html'] =
-                     "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\"
-                        'http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd'>" .
+
+                     "<!DOCTYPE html>" .
                      "<html>
                         <head>
-                         <META http-equiv='Content-Type' content='text/html; charset=utf-8'>
+                         <meta charset='utf-8' />
+                         {# prevent IE to turn into compatible mode... #}
+                         <meta http-equiv='X-UA-Compatible' content='IE=edge' />
```

Comment:
I do not know if this one is still used. It was only used by IE 8 -> IE 11 browser engine. I do not know what browser engine is used in recent Outlook versions.

```suggestion
```